src/dredd-command.coffee

```diff
@@ -54,17 +56,19 @@ class DreddCommand
 
     @argv = @optimist.argv
 
-
   # Gracefully terminate server
   stopServer: (callback) ->
-    return callback() unless @serverProcess?
+    logger.verbose('Gracefully terminating backend server process.')
+    unless @serverProcess?
+      logger.verbose('No backend server process.')
+      return callback()
 
     term = =>
-      logger.info 'Sending SIGTERM to the backend server'
+      logger.info('Sending SIGTERM to backend server process.')
       @serverProcess.kill 'SIGTERM'
 
     kill = =>
-      logger.info 'Killing backend server'
+      logger.info('Killing backend server process.')
```

Comment:
I'm afraid many developers and users of Dredd do not know anything about signals, so I'm trying to move away from these terms, at least for user-focused `info` logging (I'm fine with such expressions in `debug` etc.). Also, I'm not sure SIGKILL will be appropriate term to use when we'll start supporting Windows, would need to check.